### PR TITLE
Hudson/iframe fix

### PR DESCRIPTION
**Pull request by hudbuddy**
https://xsolla.atlassian.net/browse/LSTREAM-549

iFrames do not render with transparency if there is a mismatch between the embedding document's `color-scheme` value.
image

Interesting writeup here:
https://fvsch.com/transparent-iframes

- also fixes the height not displaying correctly

### CODE REVIEW

File: src/core/transforms/Overlay.tsx [hunk 1]
```diff
@@ -41,193 +41,202 @@ export const Overlay = {
...
-    const IFrame = React.memo(({
-      source,
-      setStartAnimation,
-    }: {
-      source: OverlaySource
-      setStartAnimation: (value: boolean) => void
-    }) => {
-      const { src, meta, height, width } = source?.sourceProps || {}
-      const iframeRef = React.useRef<HTMLIFrameElement>(null)
+    const IFrame = React.memo(
```
Comment (hudbuddy) [inline src/core/transforms/Overlay.tsx:44]: All of these lines are just formatting

File: src/core/transforms/Overlay.tsx [hunk 2]
```diff
@@ -246,7 +255,11 @@ export const Overlay = {
           duration={400}
         >
           <div
-            style={{ opacity: startAnimation ? 1 : 0 }}
+            style={{
+              opacity: startAnimation ? 1 : 0,
+              height: '100%',
```
Comment (hudbuddy) [inline src/core/transforms/Overlay.tsx:260]: only changes in the file here

**Review by maddygoround — APPROVED** [review]
This looks good!

**Comment by maddygoround** [comment]
@hudbuddy The transparency issue actually stems from Mantine's color scheme. I faced a similar issue with the alerts, and it turns out that setting the Mantine color scheme to 'none' in Studio fixed it.

/* For dark mode */
```
:root[data-mantine-color-scheme="dark"] {
  --mantine-color-scheme: none !important
}

```

**Comment by hudbuddy** [comment]
right - I was considering that approach, but I think this accomplishes the same result without having to change it. it's a weird edge case for sure

**Comment by maddygoround** [comment]
> right - I was considering that approach, but I think this accomplishes the same result without having to change it. it's a weird edge case for sure

Oh cool. I didn't notice that you already put a fix here.

**Pull request merged**